rules.md

```diff
@@ -334,15 +293,15 @@ Once you have completed the work for a Task you can make a [Claim](#26-claims) f
 2. Link to any deploy links relevant to the output (e.g. a URL for a website)
 3. Specify a valid Dash address to receive the reward, or set to [NULL](#341-null-claims) to decline the reward.
 
-Once your Claim has been created it will [processed](#435-processing-task-claims) by an admin.
+Once your Claim has been created it will [processed](#434-processing-task-claims) by an admin.
```

Comment:
will be processed (missing the "be")

Reply:
Fixed in https://github.com/dashincubator/dash-incubator-rules/pull/11/commits/c4c94a5e12f23b7efe0e79e6b4c694488486ce9b